src/app/pages/events/card/component.ts

```diff
@@ -13,8 +13,10 @@ export class EventCardComponent implements OnInit {
   public readonly datestring = 'dddd MMMM D[<span class="mobile-hide">, ]YYYY[</span>] [<br>at] hh:mma'
 
   ngOnInit(): void {
-    this.event.start = moment(this.event.start).format(this.datestring);
-    this.event.deadline = moment(this.event.deadline).format(this.datestring);
+	if (this.event && this.event.start & this.event.deadline) {
+      this.event.start = moment(this.event.start).format(this.datestring);
+      this.event.deadline = moment(this.event.deadline).format(this.datestring);
+	}
```

Comment:
good catch!

Reply:
Please feel free to overwrite my quick-guard with yours.

I just needed tests to pass in order to automate deployment :)